### PR TITLE
Promote PlayFabCommerce to PlayFab-Samples

### PR DESCRIPTION
**Pull request by timch-ms**
Key files:
[BillingManager][1]
[PlayFabMain][2]
[PlayFabOpsManager][3]
[Tutorial][4]
[Readme][5]

[1]: https://github.com/PlayFab/PlayFab-Samples/compare/timch-dev?expand=1#diff-d4429e46412ee1e9b58b7c8562b74511
[2]: https://github.com/PlayFab/PlayFab-Samples/compare/timch-dev?expand=1#diff-7147c0655c1d264522d44eb6c0cee182
[3]: https://github.com/PlayFab/PlayFab-Samples/compare/timch-dev?expand=1#diff-e62c9ef5c47c0f7abc0ceb9a99acaf10
[4]: https://github.com/PlayFab/PlayFab-Samples/compare/timch-dev?expand=1#diff-0cf9b08eb630864f98041f8ac27fb5fd
[5]: https://github.com/PlayFab/PlayFab-Samples/compare/timch-dev?expand=1#diff-c086854754501bd6b10b404e2b212941

### CODE REVIEW

File: Samples/Android/PlayFabCommerce/Readme.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+This sample uses Android's in-app billing API and PlayFab client API to demonstrate an in-app store that exchanges real money for virtual items in PlayFab Economy.
+
+## Installing the sample
+
+The app can be deployed from Android Studio to a compatible AVD, supporting SDK level 21 or above and using Google Play services (not just Google API). This is because signin with a valid Google account is required. It can also be deployed to any physical Android Device meeting the SDK requirements (5.0/Lollipop).
+
+Due to signing requirements, the deployed app may not successfully launch the real money billing flow, resulting in responseCode 4 for onPurchasesUpdated(). If this occurs, try installing the Play Store version prior to deploying the development version.
+
+The app can also be installed from the Play Store after joining the [PlayFabCommerce test group][1]. After joining (approval required, please contact [ATG Marketplace team][2]), go to the [opt-in link][3] and follow the steps to install the app on your device.
```
Comment (jameslen-atg) [inline Samples/Android/PlayFabCommerce/Readme.md]: Can we get a different alias for this?  I would prefer if atgstore wasn't publicly broadcasted.

**Comment by jameslen-atg** [comment]
Could the tutorial doc be converted to markdown?

**Comment by timch-ms** [comment]
Tutorial.docx was good for editing and wysiwyg. Markdown will require me to host the myriad images somewhere, which will present when we get the tutorial onto the web. When that happens, I can change it to markdown. In the meantime, let's throw the doc up there.

As for atgstore, i'm not keen on opening up another alias. Maybe we can reach an agreement to have such requests to devrel@playfab routed to us. I'll look into that and plan to move away from atgstore.

**Comment by timch-ms** [comment]
Changed alias in Readme to devrel@playfab

Converted tutorial to markdown

**Pull request merged**